<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>****************************************************************************
**
** Copyright (C) 2024-2025 Donna Whisnant, a.k.a. Dewtronics.
** Contact: http://www.dewtronics.com/
**
** This file is part of the KJVCanOpener Application as originally written
** and developed for Bethel Church, Festus, MO.
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU General Public License
** version 3.0 as published by the Free Software Foundation and appearing
** in the file gpl-3.0.txt included in the packaging of this file. Please
** review the following information to ensure the GNU General Public License
** version 3.0 requirements will be met:
** http://www.gnu.org/copyleft/gpl.html.
**
** Other Usage
** Alternatively, this file may be used in accordance with the terms and
** conditions contained in a signed written agreement between you and
** Dewtronics.
**
****************************************************************************</comment>
 <class>CELSBibleDatabaseSelectDlg</class>
 <widget class="QDialog" name="CELSBibleDatabaseSelectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ELSSearch Select Bible Database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblBible">
       <property name="text">
        <string>&amp;Bible</string>
       </property>
       <property name="buddy">
        <cstring>cmbBible</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBible">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkWordsOfJesusOnly">
     <property name="text">
      <string>Words of &amp;Jesus Only
(Works only with databases supporting red-letter markup)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRemoveColophons">
     <property name="text">
      <string>Remove &amp;Colophons</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRemoveSuperscriptions">
     <property name="text">
      <string>Remove &amp;Superscriptions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkIncludeBookPrologues">
     <property name="text">
      <string>Include &amp;Book Prologues (Title, Subtitle, etc.)
(Works Only on English King James Databases)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkIncludeChapterPrologues">
     <property name="text">
      <string>Include &amp;Chapter Prologues (Chapter Number, etc.)
(Works Only on English King James Databases)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbBible</tabstop>
  <tabstop>chkWordsOfJesusOnly</tabstop>
  <tabstop>chkRemoveColophons</tabstop>
  <tabstop>chkRemoveSuperscriptions</tabstop>
  <tabstop>chkIncludeBookPrologues</tabstop>
  <tabstop>chkIncludeChapterPrologues</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CELSBibleDatabaseSelectDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CELSBibleDatabaseSelectDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
